kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/Kroxylicious.java

```diff
@@ -67,11 +68,16 @@ private void deployPortPerBrokerPlain(int replicas) {
     /**
      * Needed for downstream Kroxylicious images
      */
-    private void createSecrets() {
+    public void createSecrets() {
         String configFolder = Environment.CONTAINER_CONFIG_PATH;
         SecretBuilder secretBuilder = KroxyliciousSecretTemplates.createRegistryCredentialsSecret(configFolder, deploymentNamespace);
         if (secretBuilder != null) {
-            resourceManager.createResourceWithWait(secretBuilder.build());
+            Secret secret = secretBuilder.build();
+            if (kubeClient().getClient().secrets().inNamespace(deploymentNamespace).withName(secret.getMetadata().getName()).get() != null) {
+                LOGGER.atWarn().setMessage("Skipping secrets creation as it was already created").log();
```

Comment:
Is it really a warning? if its just saying it skipped something its info to me.

Though do we really care? Should we not just let the new secret overwrite the old?

Reply:
I set that because the secret is not overwriting the old one, it is failing because it is already created. I can modify it to info.

Reply:
Ok, that's reasonable. I do wonder if we should make make it overwrite but let's just leave it for now